### PR TITLE
Added --rollback option for vic-machine configure

### LINKED ISSUE #2399: Document vic-machine configure --rollback
Per https://github.com/vmware/vic/issues/8437.

### PR DESCRIPTION
**Pull request by vidya-v**
Hi @stuclem , @wjun , and @jitinkumar2018 ,

I have updated the vic-machine configure command with the --rollback option and also added a note to run this command before rolling back a VCH upgrade.

Request your review.
Fixes #2399 

Thanks!
Vidya

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/configure_vch.md [hunk 1]
```diff
@@ -418,4 +418,16 @@ To clear the flag so that you can attempt further `vic-machine upgrade` or `vic-
...
 
-**IMPORTANT**: Before you run `vic-machine configure --reset-progress`, check in Recent Tasks in the vSphere Client that there are indeed no update or configuration operations in progress on the VCH endoint VM.
\ No newline at end of file
+**IMPORTANT**: Before you run `vic-machine configure --reset-progress`, check in Recent Tasks in the vSphere Client that there are indeed no update or configuration operations in progress on the VCH endoint VM.
+
+## Roll Back to Previous Configuration
+
+If the VCH configuration changes and you want to revert the changes, you can roll back the configuration to the previous settings by using the `--rollback` option. 
+
+Before making changes to the configuration, `vic-machine configure` takes a snapshot of the existing VCH configuration. The configuration process deletes older snapshots of any previous settings. The `--rollback` option reverts  th cofiguration of the VCH to the snapshot of the previous settings. Because `vic-machine uconfigure` only retains one snapshot, you can only use `--rollback` to revert the VCH to the configuration that immediately precedes the most recent change.
```
Comment (jitinkumar2018) [inline docs/user_doc/vic_vsphere_admin/configure_vch.md]: Hi Vidya, there are typos present

File: docs/user_doc/vic_vsphere_admin/configure_vch.md [hunk 2]
```diff
@@ -418,4 +418,16 @@ To clear the flag so that you can attempt further `vic-machine upgrade` or `vic-
...
+
+To revert the configuration to the previous settings, run `vic-machine configure` with the `--rollback` option.
+
+<pre>$ vic-machine-<i>operating_system</i> configure --rollback</pre>
+
+**Note**: If you are attempting to rollback a VCH upgrade and have to run both `vic-machine upgrade` and c`vic-machine configure` commands, you must run `vic-machine configure --rollback` before `vic-machine upgrade --rollback` in order to roll a VCH back to its previous version.
```
Comment (jitinkumar2018) [inline docs/user_doc/vic_vsphere_admin/configure_vch.md]: I think this should be rephrased to :
" If you are attempting to rollback a VCH that has been upgraded with 'vic-machine upgrade' and has undergone a configuration change with 'vic-machine configure' command, you must first rollback to the previous configuration by running 'vic-machine configure --rollback'  and then rollback to its previous VCH version by running 'vic-machine upgrade --rollback'
Or something along those lines

File: docs/user_doc/vic_vsphere_admin/configure_vch.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/configure_vch.md]: specifically "reverts  th cofiguration" and `vic-machine uconfigure`

File: docs/user_doc/vic_vsphere_admin/upgrade_vch.md [hunk 3]
```diff
@@ -73,6 +73,7 @@ For descriptions of the options that `vic-machine upgrade` includes in addition
   --id <i>vch_id</i>
   --rollback</pre>
 
+	**IMPORTANT**: Since `vic-machine configure` also takes a snapshot of the VCH, when you have to run both `vic-machine upgrade` and c`vic-machine configure` commands, you must run `vic-machine configure --rollback` before `vic-machine upgrade --rollback` in order to roll a VCH back to its previous version.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/upgrade_vch.md]: extraneous c in "c`vic-machine"

File: docs/user_doc/vic_vsphere_admin/upgrade_vch_options.md [hunk 4]
```diff
@@ -42,6 +42,8 @@ Short name: None
 
 Rolls a VCH back to its previous version, for example if upgrade failed. Before starting the upgrade process, `vic-machine upgrade` takes a snapshot of the existing VCH. The upgrade process deletes older snapshots from any previous upgrades. The `--rollback` option reverts an upgraded VCH to the snapshot of the previous deployment. Because `vic-machine upgrade` only retains one snapshot, you can only use `--rollback` to revert the VCH to the version that immediately precedes the most recent upgrade.  
 
+**IMPORTANT**: Since `vic-machine configure` also takes a snapshot of the VCH, when you have to run both `vic-machine upgrade` and c`vic-machine configure` commands, you must run `vic-machine configure --rollback` before `vic-machine upgrade --rollback` in order to roll a VCH back to its previous version.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/upgrade_vch_options.md]: extraneous c in "c`vic-machine"

**Comment by vidya-v** [comment]
Thanks, @jitinkumar2018 , @stuclem . I have made the changes.

**Comment by vidya-v** [comment]
Thanks, @stuclem !

**Pull request merged**